internal/pkg/manifest/worker_svc.go

```diff
@@ -110,16 +111,22 @@ func (q *SQSQueueOrBool) UnmarshalYAML(value *yaml.Node) error {
 
 // SQSQueue represents the configurable options for setting up a SQS Queue.
 type SQSQueue struct {
-	Retention  *time.Duration  `yaml:"retention"`
-	Delay      *time.Duration  `yaml:"delay"`
-	Timeout    *time.Duration  `yaml:"timeout"`
-	DeadLetter DeadLetterQueue `yaml:"dead_letter"`
+	Retention                 *time.Duration  `yaml:"retention"`
+	Delay                     *time.Duration  `yaml:"delay"`
+	Timeout                   *time.Duration  `yaml:"timeout"`
+	DeadLetter                DeadLetterQueue `yaml:"dead_letter"`
+	ContentBasedDeduplication *bool           `yaml:"content_based_deduplication"`
+	DeduplicationScope        *string         `yaml:"deduplication_scope"`
+	FifoThroughputLimit       *string         `yaml:"fifo_throughput_limit"`
+	HighThroughputFifo        *bool           `yaml:"high_throughput_fifo"`
+	Type                      *string         `yaml:"type"`
```

Comment:
Was there a reason why we didnt' want to group all fifo related groups together?

Here are some sample manifests:
```yaml
# option 1. standard queue
queue:
  delay: 10s
  
# option 2. fifo queue
queue:
  delay: 10s
  fifo: true

# Option 3. Advanced FIFO settings
queue:
  delay: 10s
  fifo:
     content_based_deduplication: true
     high_throughput: true
```

Reply:
Can we remove any of these fields here from the user to reduce complexity? 🤔 

For example, can we get rid of `DeduplicationScope` and `FifoThroughputLimit` if we have `high_throughput: true`?
We know that these fields must be set to respectively `messageGroup` and `perMessageGroupId` so I wonder why anyone would need to tweak them if  we already have `high_throughput` 💭